sale_product_pack/models/sale_order_line.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models, api, _
+from odoo.exceptions import UserError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    pack_type = fields.Selection(
+        related='product_id.pack_type',
+        readonly=True,
```

Comment:
I think it would be better to have it and use it. What do you think? I have changed the code to use it, check it out  https://github.com/OCA/product-pack/pull/2/files#diff-5e1acf3f740ac4f551475297b6e12b94R84

Reply:
It's talking about `readonly=True`, that it's now the default for related fields.